inlong-agent/agent-common/pom.xml

```diff
@@ -92,6 +92,11 @@
             <artifactId>commons-io</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
The liceanse and notice of the agent module need to be supplementedThe liceanse and notice of the agent module need to be supplemented